.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
+  "semi": true,
+  "trailingComma": "all",
+  "singleQuote": true,
+  "printWidth": 120,
+  "tabWidth": 2
+}
```

Comment:
can we use "useTabs": true instead of spaces, please

Reply:
I'm push new commit, please - see :)